locales/en/messages.json

```diff
@@ -4151,6 +4151,9 @@
     "osdSetupTitle": {
         "message": "OSD"
     },
+    "osdSetupNoOsdChipConfigWarning": {
+        "message": "<span class=\"message-negative\"><b>WARNING:</b></span> No OSD chip was configured. Please make sure you have <strong>osd_displayport_device</strong> set to <strong>AUTO</strong> mode."
```

Comment:
I suppose this must be done in CLI? Are other values valid different than AUTO? If yes, is better to give some more information, like:
> ... Please make sure you have <strong>osd_displayport_device</strong> set to <strong>AUTO</strong> mode (or the correct value for your craft) in CLI ."

Reply:
Yes @McGiverGim good point, looking at this https://github.com/betaflight/unified-targets/pull/270 seems that now it depends of the board type.

Reply:
Done @McGiverGim ,please tell me if it need English corrections